lib/node_modules/@stdlib/_tools/eslint/rules/vars-order/lib/main.js

```diff
@@ -122,6 +107,87 @@ function main( context ) {
 				prevLength = name.length;
 			}
 		}
+
+		/**
+		* Reports the error message.
+		*
+		* @private
+		* @param {string} msg - error message
+		* @param {Object} loc - lines of code (object with `start` and `end` properties)
+		*/
+		function report( msg, loc ) {
+			context.report({
+				'node': null,
+				'message': msg,
+				'loc': loc,
+				'fix': fix
+			});
+		}
+
+		/**
+		* Fixes the lint error by reordering the variable declarations inside of the function.
+		*
+		* @private
+		* @param {Function} fixer - ESLint fixer
+		* @returns {(Object|null)} fix or null
+		*/
+		function fix( fixer ) {
```

Comment:
Looking at this function, I believe it can be moved to a higher scope. Ditto for `report`.